kit/cli/viper.go

```diff
@@ -63,40 +63,40 @@ func NewCommand(p *Program) *cobra.Command {
 		},
 	}
 
-	viper.SetEnvPrefix(strings.ToUpper(p.Name))
-	viper.AutomaticEnv()
+	v.SetEnvPrefix(strings.ToUpper(p.Name))
+	v.AutomaticEnv()
```

Comment:
Side note: I noticed `cmd/influx` duplicates some of this logic, should I open a tech-debt ticket to port it over to `kit/cli`?

Reply:
Yes, I think that is a good idea

Reply:
#20013